ravenframework/SupervisedLearning/DynamicModeDecompositionControl.py

```diff
@@ -415,7 +556,7 @@ def _evaluateMatrices(self, X1, X2, U, Y1, rankSVD):
     # SVD
     uTrucSVD, sTrucSVD, vTrucSVD = mathUtils.computeTruncatedSingularValueDecomposition(omega, rankSVD, False, False)
     # Find the truncation rank triggered by "s>=SminValue"
-    rankTruc = sum(map(lambda x : x>=1e-6, sTrucSVD.tolist()))
+    rankTruc = sum(map(lambda x : x>=np.max(sTrucSVD)*1e-9, sTrucSVD.tolist()))
```

Comment:
I would suggest to define a self variable for the tolerance here, i.e., 1e-9.

Reply:
you are right. Sorry...adding it.